programmers/난이도별/level02.멀쩡한_사각형/dkdlelk99.py

```diff
@@ -0,0 +1,27 @@
+# 출처 https://programmers.co.kr/learn/courses/30/lessons/62048
+# input W : 가로 길이, H : 세로 길이 (1억 이하의 자연수)
+# output 멀쩡한 사각형의 갯수
+# 가로 길이 W, 세로 길이 H인 사각형을 대각선으로 자를 때, 가로 세로가 1cm인 사각형이 몇개인가?
+
+import math
+def solution(w,h):
+    answer = 0
+    all = w * h
+    x, y = w, h # x는 최대 공약수
+    while y:
+        x, y = y, x % y
+    # 수학을 이용한 간편한 풀이
+    # answer = (w//x - 1) + (h//x - 1) + 1
+    # return all - answer * x
+    if w == h:
+        return all - w
+    elif h > w:
```

Comment:
이전의 조건문에 들어가면 리턴되니까 else문은 붙이지 않아도 되지 않을까요??
```suggestion
    if h > w:
```

Reply:
그러네요! 좋은 의견 감사합니다 :D